src/engine/UIManager.cxx

```diff
@@ -223,6 +225,11 @@ void UIManager::init()
           {
             m_uiGroups[groupID].push_back(uiElement.get());
           }
+
+          if (uiElement->getUiElementData().elementID == "slider")
+          {
+            m_slider = dynamic_cast<Slider *>(uiElement.get());
```

Comment:
Nevertheless, this is *very* hacky.  Take a look at `void UIManager::setCallbackFunctions()`

There's a function that checks for specific IDs and then adds callback functions for it.

Reply:
I'll take a look into this one.